File: Homework/Objective-C/HW7/HW7/Runner.m
//
//  Runner.m
//  HW7
//
//  Created by Руслан on 18.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

#import "Runner.h"

@implementation Runner

- (NSString*) run {
    return [NSString stringWithFormat:@"Runner %@ is running", self.name];
}

- (NSString*) laugh {
    return [NSString stringWithFormat:@"Runner %@ is laughing", self.name];
}

@end

File: Homework/Swift/HW7/HW7/Runner.swift
//
//  Runner.swift
//  HW7
//
//  Created by Руслан on 19.09.16.
//  Copyright © 2016 Ruslan Timchenko. All rights reserved.
//

import Foundation

class Runner: Runners {
    
    let name: String
    
    init(name: String) {
        self.name = name
    }
    
    func run() -> String {
        return "Runner \(self.name) is running"
    }
    
    func laugh() -> String {
        return "Runner \(self.name) is laughing"
    }

}
